src/slides/1-demos/chevres/responsive-final.css

```diff
@@ -0,0 +1,82 @@
+.content {
+	display: grid;
+	align-items: center;
+	grid-template-columns: min(25%, 7em) 1fr;
+	column-gap: 1rem;
+
+	img {
+		width: 100%;
+		aspect-ratio: 1;
+		object-fit: cover;
+		border-radius: 0.75rem;
+	}
+
+	h1 {
+		font-size: 1rem;
+		margin: 0 0 0.25em;
+	}
+
+	p {
+		font-size: 0.75rem;
+		margin: 0;
+	}
+}
+
+article {
+	container: article / inline-size;
+}
+
+@container article (max-width: 340px) {
+	.content {
+		grid-template: auto auto / 1fr;
+		row-gap: 0.5rem;
+
+		img {
+			aspect-ratio: 320 / 175;
+		}
+
+		h1 {
+			font-size: 1.25rem;
+		}
+
+		p {
+			font-size: 0.875rem;
+		}
+	}
+}
+
+@container article (min-width: 520px) {
+	.content {
+		grid-template: auto / 1fr;
+		position: relative;
+
+		&::before {
+			content: "";
+			position: absolute;
+			inset: 0;
+			border-radius: 0.75rem;
+			background: linear-gradient(70deg, black 350px, transparent 700px),
+				linear-gradient(to top, black -20%, transparent 30%);
+		}
```

Comment:
<!-- Praises highlight something positive. Try to leave at least one of these comments per review (if it exists :^)  -->
**👏 praise:** ‏Beautiful 🔥